src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -1020,3 +1020,17 @@ def addTier1Skim(config, skimName, dataTier, primaryDataset, cmsswVersion,
 
     datasetConfig.Tier1Skims.append(skimConfig)
     return
+
+def setStreamMem(config, *streamid, **streamidvalue):
```

Comment:
I'm not sure why need *streamid to acomplish this. What is the use of having dictionary keys named "arg1", "arg2", etc.? there are never going to bwe recognized in RunConfigAPI